src/components/Popup.js

```diff
@@ -3,18 +3,18 @@ export default class Popup {
         this._popup = document.querySelector(popupSelector);
     }
 
-    open() {
+    open(){
         this._popup.classList.add("modal_opened");
-        document.addEventListener("keydown", this._handleEscClose);
+        document.addEventListener("keydown", (e)=>{
+           if(e.key === "Escape"){
+            this.close();
+           }
+           
+        });
     };
     close() {
         this._popup.classList.remove("modal_opened");
-        document.removeEventListener("keydown", this._handleEscClose);
-    }
-    _handleEscClose = (e) => {
-        if (e.key === "Escape" || e.key === "Esc") {
-            this.close();
-        }
+        //document.removeEventListener("keydown", this._handleEscClose);

```

Comment:
Please, return the right code back. You had `document.removeEventListener("keydown", this._handleEscClose);` here


The checklist says: an event listener for closing the popup by pressing Esc is added when the popup is opened, and is removed when it's closed.